visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -129,15 +135,28 @@ void MotionPlanningFrame::computePlanButtonClicked()
 void MotionPlanningFrame::computeExecuteButtonClicked()
 {
   if (move_group_ && current_plan_)
-    move_group_->execute(*current_plan_);
+  {
```

Comment:
Do u mean `computeExecuteButtonClicked()` and `computeStopButtonClicked()` seems to take few time to execute, and it seems not necessary to run it in a separate thread?